dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeFactoryImpl.java

```diff
@@ -323,10 +325,9 @@ private ContentType dbByVar(String var) throws DotDataException {
   }
 
   private ContentType dbSaveUpdate(final ContentType saveType) throws DotDataException {
-
-
-    ContentTypeBuilder builder =
-        ContentTypeBuilder.builder(saveType).modDate(DateUtils.round(new Date(), Calendar.SECOND));
+    final ContentTypeBuilder builder = ContentTypeBuilder
```

Comment:
I would add some ITs to this logic, it can be at factory or api level

Reply:
@nollymar This is tested in the postman ContentTypeResourceTests.json postman test.

Reply:
It is ok to have the curl test, but they aren't meant to replace integration tests, especially when we have heavy logic in the API or Factory class

Reply:
IT created.